internal/proto/controller/api/resources/groups/v1/group.proto

```diff
@@ -14,19 +14,19 @@ option go_package = "github.com/hashicorp/boundary/sdk/pbs/controller/api/resour
 
 message Member {
   // Output only. The ID of the member.
-  string id = 10; // @gotags: `class:"public"`
+  string id = 10; // @gotags: `class:"public" eventstream:"observation"`
 
   // Output only. The Scope ID of the member.
-  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public" eventstream:"observation"`
 }
 
 // Group contains all fields related to a Group resource
 message Group {
   // Output only. The ID of the Group.
-  string id = 10; // @gotags: `class:"public"`
+  string id = 10; // @gotags: `class:"public" eventstream:"observation"`
 
   // The ID of the scope of which this Group is a part.
-  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public" eventstream:"observation"`
 
   // Output only. Scope information for this Group.
   resources.scopes.v1.ScopeInfo scope = 30;
```

Comment:
What does it mean to add this tag to only some fields in a message?

Reply:
These fields are specifically chosen for the telemetry pipeline and for the product team where they are not customer-specified values

Reply:
Maybe I'm missing something - what happens to the fields that are tagged? What values can the `eventstream` tag take?

Reply:
we read and detect them inside observations_event and we will add them to the observation pipeline. for now just an observation value but we might have another type later down the line.

Reply:
So when reviewing this you just want me to check that the right fields are tagged?

Reply:
Exactly, Thank you 🙏

Reply:
and for sure If I messed up in any other parts 😄